containers/app/entrypoint.sh

```diff
@@ -18,6 +18,11 @@ if [ -z "$SANDBOX_USER_ID" ]; then
   exit 1
 fi
 
+if [ -z "$WORKSPACE_MOUNT_PATH" ]; then
+  # This is set to /opt/workspace by default. But if the user isn't mounting, we want to unset it so that OpenHands doesn't mount at all
```

Comment:
```suggestion
  # This is set to /opt/workspace in the Dockerfile. But if the user isn't mounting, we want to unset it so that OpenHands doesn't mount at all
```